types/tna/tna/index.ts

```
@@ -1,6 +1,11 @@
 import { DateInfo } from '@/types/commons/dateInfo';
 import { StatusBadgeTheme } from '@/components/common/statusBadge';
-import { Meta } from '@/store/server/features/okrPlanningAndReporting/interface';
+// import { Meta } from '@/store/server/features/okrPlanningAndReporting/interface'; // Commented out due to missing module
+
+// Placeholder type since module is missing
+export interface Meta {
+  // Placeholder interface - no actual properties
+}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)